### ISSUE TITLE
tuterm: error: fake home can only be created once!

### ISSUE THREAD

**Issue by ajeetdsouza** [veracioux/tuterm#14]
I'm trying to follow the `ls_tutorial` from the README, but I get the following error:

```
$ tuterm ls_tutorial --mode demo
tuterm: error: fake home can only be created once!
Aborting.
```

**Comment by veracioux** [comment]
Thanks. I made the change and then forgot to update the examples. I'm on it.

**Comment by veracioux** [comment]
@ajeetdsouza Should be fixed by 31f8d8d00e69656c88cc03b7d25f34d75bb54ab9

**Comment by ajeetdsouza** [comment]
Thanks!
